docs/en/engines/table-engines/mergetree-family/annindexes.md

```diff
@@ -92,7 +99,7 @@ CREATE TABLE t
 (
   id Int64,
   number Tuple(Float32, Float32, Float32),
-  INDEX x number TYPE annoy(T) GRANULARITY N
+  INDEX x number TYPE annoy(Trees, DistanceName) GRANULARITY N
```

Comment:
`x` is quite confusing as index name, maybe `ann_index_name` or `ann_number_index`?